src/components/mgt-tasks/mgt-tasks.ts

```diff
@@ -1052,16 +1081,18 @@ export class MgtTasks extends MgtTemplatedComponent {
           >
             ${taskCheck}
           </span>
-          <div class="TaskDetailsContainer">
+          <div class="TaskDetailsContainer ${this._mediaQuery}">
             ${taskDetails}
           </div>
+          ${taskOptions}
+
+          <div class="Divider"></div>
         </div>
-        ${taskOptions}
       </div>
     `;
   }
 
-  private renderAssignedPeople(task: ITask) {
+  private renderAssignedPeople(task: ITask, style: string = 'TaskDetail TaskAssignee') {
```

Comment:
please change this to remove the style (should be class anyway) second parameter and use a classMap instead. Ex:

```js
    const classes = {
      TaskDetail: task !== null,
      TaskAssignee: task !== null,
      NewTaskAssignee: task === null
    };

    return html`
      <mgt-flyout
        class=${classMap(classes)}
```